pkg/network/istio/ports.go

```diff
@@ -42,3 +43,9 @@ func ReservedPorts() []string {
 		fmt.Sprint(EnvoyPrometheusTelemetryPort),
 	}
 }
+
+func NonProxiedPorts() []int32 {
```

Comment:
Why `int32` and not a regular `int`?

Reply:
Because `port.Port` is `int32` too, but now I see that we're casting it to int already. I'll use int then.
Btw, I kept the ReservedPorts to return `[]string`, because I'm joining the ports and I would need to convert it to slice of strings anyway

Reply:
[int32 -> int](https://github.com/kubevirt/kubevirt/compare/bfd108896186472bff50cdc86bdfb0ef56ec7ce3..17b059dbf2947eb046da101d448955c6ec511bc4)